partner_contact_access_link/README.rst

```diff
@@ -71,6 +71,7 @@ Contributors
 * Jairo Llopis (`Tecnativa <https://www.tecnativa.com/>`__)
 * Harald Panten (`Sygel Technology <https://www.sygel.es/>`__)
 * Tharathip Chaweewongphan <tharathipc@ecosoft.co.th>
+* Bob Leers (`Nova Code <https://www.novacode.nl/>`__)
```

Comment:
Remove the changes in the README file.
You should change the contributors file instead as is mentioned here https://github.com/OCA/maintainer-tools/tree/master/template/module

Reply:
Done @emagdalenaC2i 
Let me know if any issues.

Reply:
@bobslee Sorry but I can't see the difference with this change
Please, could you explain it?

Reply:
@emagdalenaC2i I squashed the commits. So there should now be only 1 commit:
4f2ab4ef88e1fe16e5c9a1a4adc391f9dde92cac